### ISSUE TITLE
Error calling kind script

### ISSUE THREAD

**Issue by jacobdotcosta** [snowdrop/k8s-infra#308]
```
bash <(curl -s -L https://raw.githubusercontent.com/snowdrop/k8s-infra/main/kind/kind.sh)                                                                                                  INT ✘  18:01:36

/dev/fd/13: line 877: [: -v: unary operator expected
/dev/fd/13: line 881: [: -v: unary operator expected
/dev/fd/13: line 885: [: -v: unary operator expected
/dev/fd/13: line 889: [: ==: unary operator expected

Welcome to our

   _____                                  _
  / ____|                                | |
 | (___    _ __     ___   __      __   __| |  _ __    ___    _ __
  \___ \  | '_ \   / _ \  \ \ /\ / /  / _  | |  __|  / _ \  | \ _ \
  ____) | | | | | | (_) |  \ V  V /  | (_| | | |    | (_) | | |_) |
 |_____/  |_| |_|  \___/    \_/\_/    \__,_| |_|     \___/  |  __/
                                                            | |
                                                            |_|
Script to create/delete a Kubernetes cluster using kind like a :
- (secured) docker container registry
- ingress controller (nginx, kourier)

/dev/fd/13: line 225: [: ==: unary operator expected
✔ Pre requisites check passed!
```

**Comment by aureamunoz** [comment]
I have also a problem with it even if it moves forward and creates the cluster, I have a problem regarding the internal registry. This is the output:
```
./kind.sh install --delete-kind-cluster                                                                                                                                        125 ✘  18s  15:15:24 
./kind.sh: line 877: [: -v: unary operator expected
./kind.sh: line 881: [: -v: unary operator expected
./kind.sh: line 885: [: -v: unary operator expected
./kind.sh: line 889: [: -v: unary operator expected

Welcome to our
                                                                   
   _____                                  _                        
  / ____|                                | |                       
 | (___    _ __     ___   __      __   __| |  _ __    ___    _ __  
  \___ \  | '_ \   / _ \  \ \ /\ / /  / _  | |  __|  / _ \  | \ _ \ 
  ____) | | | | | | (_) |  \ V  V /  | (_| | | |    | (_) | | |_) |
 |_____/  |_| |_|  \___/    \_/\_/    \__,_| |_|     \___/  |  __/ 
                                                            | |    
                                                            |_|    
Script to create/delete a Kubernetes cluster using kind like a : 
- (secured) docker container registry
- ingress controller (nginx, kourier)

✔ Pre requisites check passed!
! Deleting Kind cluster...
✔ Removing kind cluster (kind)...
✔ Removing kind registry container...
✔ ...done!
! Checking if kind cluster exists...
Creating cluster "kind" ...
DEBUG: docker/images.go:58] Image: kindest/node:v1.26.3@sha256:61b92f38dff6ccc29969e7aa154d34e38b89443af1a2c14e6cfbd2df6419c66f present locally
 ✓ Ensuring node image (kindest/node:v1.26.3) 🖼
 ✓ Preparing nodes 📦  
 ✓ Writing configuration 📜 
 ✓ Starting control-plane 🕹️ 
 ✓ Installing CNI 🔌 
 ✓ Installing StorageClass 💾 
Set kubectl context to "kind-kind"
You can now use your cluster with:

kubectl cluster-info --context kind-kind

Thanks for using kind! 😊
configmap/local-registry-hosting created
716710aa3615603f3c3c7a5147013d833d0eecc8f846e5f8a8a88612c56c0ed1
docker: Error response from daemon: Ports are not available: exposing port TCP 0.0.0.0:5000 -> 0.0.0.0:0: listen tcp 0.0.0.0:5000: bind: address already in use.

```

**Comment by aureamunoz** [comment]
The problem is that it is not getting to expose the ports, so the registry container is created but not running: 

![Captura de pantalla 2023-05-17 a las 15 54 06](https://github.com/snowdrop/k8s-infra/assets/6627952/d9392412-c9c9-4240-ac97-f4578af00361)
